### PR TITLE
Add hapi-recaptcha plugin to the list

### PR DESCRIPTION
**Pull request by cristobal151**
A simple google's recaptcha plugin for hapi.

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -389,6 +389,10 @@ exports.categories = {
         tv: {
             url: 'https://github.com/hapijs/tv',
             description: 'Interactive debug console plugin for hapi servers'
+        },
+        'hapi-recaptcha': {
+            url: 'https://github.com/cristobal15/hapi-recaptcha',
+            description: 'Google's reCaptcha for hapi'
```
Comment (arb) [inline lib/plugins.js]: This isn't valid JS.

Comment (nlf) [inline lib/plugins.js]: what he means by that is you need to escape the single quote character like `Google\'s`

File: lib/plugins.js [hunk 2]
```diff
@@ -389,6 +389,10 @@ exports.categories = {
         tv: {
             url: 'https://github.com/hapijs/tv',
             description: 'Interactive debug console plugin for hapi servers'
+        },
+        'hapi-recaptcha': {
+            url: 'https://github.com/cristobal15/hapi-recaptcha',
```
Comment (gergoerdosi) [inline lib/plugins.js]: The URL isn't valid either, your username is wrong.

**Comment by cristobal151** [comment]
Sorry for the mistakes.
I already made changes in the url and description.

Thanks.

**Pull request merged**